cfpb/cfgov-refresh
# URL changes to match new site map

Moves /blog, /newsroom, and /about-us to their intended URL, per @schaferjh document

Initial approach was to fix this via server configuration, but this seems like a more sustainable way of doing it...

Also adds some fin-ed-resources URL's that were missing, like the new-ish money-as-you-grow.

## Changes

- change urls.py so that blog,  newsroom, and the-bureau are under /about-us
- adds missing URL's for fin-ed-resources
- change all templates and sheerlike settings to reference the new URL's
- redirects the old URL's to the new ones
- move the-bureau sheer templates under about-us

## Testing

- observe that all links go to new blog and newsroom urls, and that the blog and newsroom (including filter, etc) performs as before.

To test the fin-ed-resources stuff:

- checkout out fin-ed-resources to a sibling directory of cfgov-refresh
- in that directory, `npm install` and `grunt`
- back in cfgov-refresh, `cfgov/manage.py sheer_index`
- `runserver` and observe that you can browse to localhost:8000/money-as-you-grow/

## Review

- @kurtw
- @kave
- @richaagarwal


## Checklist
